_config.yml

```diff
@@ -14,7 +14,7 @@ site_title: "Google Summer of Code Guides"
 company_name: The Google Summer of Code Guides Authors
 # this appears in the footer
 
-copyright_year: 2009-2017
+copyright_year: 2009-2020
```

Comment:
2020 is incorrect, as no changes have been made to the content in 2020.

A better change is to just change it to 2009.   It's the initial year of creation that matters.